docs/contributor/04-10-testing-strategy.md

```diff
@@ -4,24 +4,24 @@
-- `Serverless verify / git-auth-integration-test (push)` - Runs the `GitHub` and `Azure DevOps` API and authentication integration test suite for Serverless. For the configuration, see the [serverless-verify.yaml](https://github.com/kyma-project/serverless/blob/main/.github/workflows/serverless-verify.yaml) file.
-- `Operator verify / gardener-integration-test (push)` - Checks the installation of the Serverless module in the Gardener shoot cluster and runs basic integration tests of Serverless. For the configuration, see the [operator-verify.yaml](https://github.com/kyma-project/serverless/blob/main/.github/workflows/operator-verify.yaml) file.
+- `integration tests (push) / operator-integration-test` - Runs the create/update/delete Serverless integration tests in k3d cluster. For the configuration, see the [integration-tests-push.yaml](https://github.com/kyma-project/serverless/blob/main/.github/workflows/integration-tests-push.yaml) file.
+- `integration tests (push) / serverless-integration-test` - Runs the basic functionality integration and the `tracing`, `api-gateway`, and `cloud-event` contract compatibility integration test suite for the Serverless in a k3d cluster. For the configuration, see the [integration-tests-push.yaml](https://github.com/kyma-project/serverless/blob/main/.github/workflows/integration-tests-push.yaml) file.
+- `integration tests (push) / git-auth-integration-test` - Runs the `GitHub` and `Azure DevOps` API and authentication integration test suite for Serverless. For the configuration, see the [integration-tests-push.yaml](https://github.com/kyma-project/serverless/blob/main/.github/workflows/integration-tests-push.yaml) file.
+- `integration tests (push) / gardener-integration-test` - Checks the installation of the Serverless module in the Gardener shoot cluster and runs basic integration tests of Serverless. For the configuration, see the [integration-tests-push.yaml](https://github.com/kyma-project/serverless/blob/main/.github/workflows/integration-tests-push.yaml) file.
+- `upgrade tests / operator-upgrade-test` - Runs the upgrade integration test suite and verifies if the latest release can be successfully upgraded to the new (`main`) revision. For the configuration, see the [upgrade-tests.yaml](https://github.com/kyma-project/serverless/blob/main/.github/workflows/upgrade-tests.yaml) file.
+- `upgrade tests / serverless-upgrade-test` - Runs the basic functionality integration and the `tracing`, `api-gateway`, and `cloud-event` contract compatibility integration test suite for the Serverless in a k3d cluster after upgrading from the latest release to the actual revision to check if the serverless component is working properly after the upgrade. For the configuration, see the [upgrade-tests.yaml](https://github.com/kyma-project/serverless/blob/main/.github/workflows/upgrade-tests.yaml) file.
```

Comment:
Serverless component or module?
```suggestion
- `upgrade tests / serverless-upgrade-test` - Runs the basic functionality integration and the `tracing`, `api-gateway`, and `cloud-event` contract compatibility integration test suite for Serverless in a k3d cluster after upgrading from the latest release to the actual revision to check if the Serverless module is working properly after the upgrade. For the configuration, see the [upgrade-tests.yaml](https://github.com/kyma-project/serverless/blob/main/.github/workflows/upgrade-tests.yaml) file.
```

Reply:
"component". In this context we say about part of Serverless module which manage functions and is historically called ... Serverless :(

Reply:
Ok, got it. I've updated my comment :)